tracker/templates/home.html

```diff
@@ -0,0 +1,40 @@
+{%- extends "base.html" -%}
+{% block content %}
+
+    <h1>Arch Linux Security</h2>
+
+    For Arch Linux security is paramount, and a dedicated team of staffers and
+    volunteers helps with triaging, prioritizing, confirming, patching and
+    notifying about vulnerabilities in Arch Linux components as well as
+    packages provided in the official repositories.
+
+    <h2>Reporting Vulnerablities in Arch Linux Components or Infrastructure</h3>
+
+    If you have discovered a vulnerability in packages related to core
+    components or in Arch Linux infrastructure don't hesitate to reach out 
+    to <a href="mailto:security@archlinux.org">security@archlinux.org</a> to
+    discuss remediation. You can encrypt your email to the following gpg keys.
+    <!-- FIXME: we'll have to figure this one out on the backend -->
+    <ul>
+       <li><a href="https://www.archlinux.org/people/developers/#anthraxx">Levente Polyak</a>
+           <tt>E240 B57E 2C46 30BA 768E  2F26 FC1B 547C 8D81 72C8</tt></li>
+
+       <li><a href="https://www.archlinux.org/people/support-staff/#rgacogne">Remi Gacogne</a>
+           <tt>A4CB EA79 7489 8599 195E  4FEC 46EC 46F3 9F3E 2EF1</tt></li>
+
+       <li><a href="https://www.archlinux.org/people/developers/#allan">Allan McRae</a>
+           <tt>6645 B0A8 C700 5E78 DB1D  7864 F99F FE0F EAE9 99BD</tt></li>
+    </ul>
+
+    <h2>Vulnerabilities in Upstream Packages</h3>
+
+    Our <a href="https://security.archlinux.org/issues/">CVE tracker</a> can help you
+    review the vulnerabilities that affect the packages we provide. If you
+    think a CVE that affects a package is not listed in there, do not hesitate
+    to reach out to us in irc on <tt>#archlinux-security</tt>.
+
+    <h2>Securing your Arch Linux installation</h3>
+
+    Consider checking this 
+    <a href="https://wiki.archlinux.org/security/">Wiki page for more information</a>
```

Comment:
Should be https://wiki.archlinux.org/index.php/Security the other page gives a 403.